doc/radosgw/admin.rst

```diff
@@ -473,6 +473,157 @@ commands. ::
    be restarted for the changes to take effect.
 
 
+Rate Limit Management
+=====================
+
+The Ceph Object Gateway enables you to set rate limits on users and buckets. 
+rate limit include the maximum number of read ops and write ops per minute
+and how many write bytes or read bytes per minute a user or bucket can make.
+Requests that are using GET or HEAD method in the REST request considered as read, otherwise it is write.
+Every Object Gateway will track after the user/bucket usage itself and will not share those metrics with other gateways.
+That means that the desired limits configured should be divide by the number of active Object Gateways.
+if userA should be with a limit 10 ops per minute and there are 2 Object Gateways in the cluster,
+the limit over userA should be 5 (10 ops per minute / 2 RGWs).
+
```

Comment:
please add a note here, explaining that if the requests are not balanced between RGWs, the rate limit may be underutilized.